.github/workflows/deploy-platform-staging.yml

```diff
@@ -0,0 +1,83 @@
+name: deploy-platform-staging
+
```

Comment:
Just wondering, aren't we able to reuse the same configuration file for multiple environments? @danielogen 
But I guess it is not yet possible as seen here: https://github.com/actions/starter-workflows/issues/245